src/pages/index.js

```diff
@@ -151,18 +256,23 @@ profileModalEditButton.addEventListener("click", openEditProfileModal);
 /*                            Initializing Web Page                           */
 /* -------------------------------------------------------------------------- */
 
-/* --------------------------- Web Page Secttions --------------------------- */
+/* --------------------------- Web Page Sections --------------------------- */
 
 const cardsListSection = new Section(
   {
     items: initialCards,
     renderer: (data) => {
       const card = createCard(data);
-      cardsListSection.addItem(card);
+      cardsListSection.addInitialItems(card);
```

Comment:
Needs correction: you don't need `initialCards` array anymore, you should initialize the cards only after receiving them:

```js
let cardsListSection;

api
  .getCards()
   .then((cards) => {
       cardsListSection = new Section(..);
    })
  .catch((err) => console.error(err));
```